mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,159 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
```

Comment:
I'd remove color gamut and add in max brightness. 

The core considerations for high dynamic range are more focused on brightness: high maximum brighness, high contrast ratio, and high bit depth. Similar to contrast ratio, the meaning of "high" for these other attributes is also not standardized. Something like >= 400 nits and >= 10 bits per channel, but I wouldn't write those values into the spec. Combined (for all channels) bit depth >= 24 is probably fine given the history/current spec for that attribute. 

Color gamut is distinct and well covered by the existing attribute. Its true that we will usually find HDR displays to have one of these color gamuts, but its not inherent in the definition of HDR and we'll be more future proof being less specific about the color gamuts.